### PR TITLE
dts: add overlay for pitft22

### PR DESCRIPTION
**Pull request by agherzan**
Add the pitft22 overlay from adafruit Adafruit-Pi-Kernel-o-Matic repo:
https://github.com/adafruit/Adafruit-Pi-Kernel-o-Matic

Signed-off-by: Petter Mabäcker petter@technux.se
Signed-off-by: Andrei Gherzan andrei@resin.io

### CODE REVIEW

**Comment by pelwell** [comment]
Add a README entry and I'll merge it.

**Comment by agherzan** [comment]
Added. Thanks.

File: arch/arm/boot/dts/overlays/Makefile [hunk 1]
```diff
@@ -51,6 +51,7 @@ dtbo-$(RPI_DT_OVERLAYS) += pi3-disable-bt.dtbo
 dtbo-$(RPI_DT_OVERLAYS) += pi3-miniuart-bt.dtbo
 dtbo-$(RPI_DT_OVERLAYS) += piscreen.dtbo
 dtbo-$(RPI_DT_OVERLAYS) += piscreen2r.dtbo
+dtbo-$(RPI_DT_OVERLAYS) += pitft22-overlay.dtbo
```
Comment (pelwell) [inline arch/arm/boot/dts/overlays/Makefile:54]: This should be `pitft22.dtbo`.

**Comment by pelwell** [comment]
Everything looks good apart from the Makefile - remove the `-overlay`.

**Comment by pelwell** [comment]
It's OK - I've patched the commit and merged it myself.

**Comment by agherzan** [comment]
I was just about to push the fix. Thanks and sorry for the mess. Will do it right next time. Cheers @pelwell

**Comment by pelwell** [comment]
No problem.

**Pull request closed without merging**